CorsixTH/CMakeLists.txt

```diff
@@ -277,7 +277,12 @@ if(NOT USE_SOURCE_DATADIRS)
     install(CODE "
       INCLUDE(BundleUtilities)
       SET(BU_CHMOD_BUNDLE_ITEMS ON)
-      FIXUP_BUNDLE(${CMAKE_INSTALL_PREFIX}/CorsixTH.app \"\" \"\")
+      FIXUP_BUNDLE(\"${CMAKE_INSTALL_PREFIX}/CorsixTH.app\" \"\" \"\")
```

Comment:
Is adding quotes here needed as that's CMake code and should be handled fine? They were needed on line 284 because bash doesn't handle unquoted filepaths with spaces.

Someone who knows more abotu CMake would need to comment on that.

Reply:
> -- Installing: /Applications/test path/CorsixTH.app/Contents/Resources/LICENSE.txt
> -- fixup_bundle
> --   app='/Applications/test'
> --   libs='path/CorsixTH.app'
> --   dirs=''
> --   ignoreItems=''
> -- warning: *NOT* handled - directory/file does not exist...
> CMake Error at /usr/local/Cellar/cmake/3.17.3/share/cmake/Modules/BundleUtilities.cmake:988 (message):
>   error: fixup_bundle: not a valid bundle
> Call Stack (most recent call first):
>   cmake_install.cmake:62 (FIXUP_BUNDLE)
> -- fixup_bundle: done

Without that, I get this error where CMake has the same problem.

Reply:
Ah yes that makes sense. It might be worthwhile as a separate PR to go through and quote all filepath variables to prevent this anywhere else in CMake.